core/Plugin/Menu.php

```diff
@@ -30,16 +30,6 @@
  */
 class Menu
 {
-    protected $module = '';
```

Comment:
In that case the problem is with `urlForAction()` where `$this->getModule()` would simply be put in a variable, there is no need for a cache for `getModule()`. A quick look in blackfire shows that `get_class()` doesn't even appear in the profiling, it's no performance concern at all.